### PR TITLE
Fix the workflow by adding the version resolve step

### PR DESCRIPTION
**Pull request by Link-**

### CODE REVIEW

**Review by rentziass — DISMISSED** [review]

File: .github/workflows/publish-arc.yaml [hunk 1]
```diff
@@ -71,22 +81,13 @@ jobs:
             echo "RELEASE_TAG_NAME=${{ github.event.inputs.release_tag_name }}" >> $GITHUB_ENV
           fi
 
-      - name: Resolve push to registries
```
Comment (mumoshu) [inline .github/workflows/publish-arc.yaml:74]: To be extra sure, we still need this step regardless of we set up the `VERSION` env for `make github-release` ☺️

Comment (Link-) [reply .github/workflows/publish-arc.yaml:74]: 100% ! I renamed the variables to reduce redundancy

Comment (mumoshu) [reply .github/workflows/publish-arc.yaml:74]: Ah! It looks even more nicer now!

**Review by mumoshu — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**